Devscord.DiscordFramework/Middlewares/Factories/UserRoleFactory.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Linq;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Discord.Rest;
+using Discord.WebSocket;
+
+namespace Devscord.DiscordFramework.Middlewares.Factories
+{
+    public class UserRoleFactory
+    {
+        public UserRole Create(SocketRole socketRole)
```

Comment:
unikałbym takich zależności z discord.net

Reply:
W jaki sposób, skoro ta klasa właśnie ma zamienić z zewnętrznych klas na nasze?

Reply:
mam na myśli parametry metod, a nie ciała

Reply:
ta klasa rzeczywiście ma to robić i jeśli się nie da zrobić czegoś inaczej to twórzmy zależności 

ale jeśli da się coś zrobić bez używania obiektów z biblioteki
bo da się przykładowo przekazywać ID jako int/long
to dobrze robić to właśnie w taki sposób